composer.json

```diff
@@ -2,9 +2,9 @@
     "name": "php-deal/framework",
     "description": "Design by Contract framework for PHP",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.0",
         "goaop/framework": "~2.0",
-        "beberlei/assert": "^2.4"
+        "beberlei/assert": "~3.0"
```

Comment:
I just looked into the code. And this assertion library is only used for tests.

Could you please move this package to `require-dev` with the new provided version.
And also add the `suggest` part in the `composer.json` to suggest this library for everyone using php-deal.

Reply:
@icanhazstring sure, but `beberlei/assert` requires php v7 and above...

Reply:
I will open a PR to bump min goaop to `~7.0` then we can bump this as well and merge your PR

Reply:
@panosru since the PR to bump min version of goaop to ~7.0 should be merged soon.
We can merge this one as welll.

- bump min version to ~7.0
- add `beberlei/assert` as `require-dev` and `suggest`